src/components/App.js

```diff
@@ -0,0 +1,16 @@
+import React, { PureComponent } from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+class App extends PureComponent {
+  render() {
+    return (
+      <div>
```

Comment:
- Use a fragment in other to create an efficient render and no creation of extra node. `<></>`